src/patternfly/components/Check/check.scss

```
@@ -52,7 +52,7 @@
 
 .#{$check}__body {
   grid-column: 2;
-  margin-top: var(--#{$check}__body--MarginTop);
+  transform: translateY(var(--#{$check}__body--TranslateY));
```

```suggestion
  transform: translateY(var(--#{$check}__body--TranslateY)); // inline-block causes a misalignment that this fixes
```
(something like that; i.e. why the heck are we manually adjusting this)